src/FsAutoComplete.Core/TypedAstUtils.fs

```diff
@@ -1,4 +1,4 @@
-///Original code from VisualFSharpPowerTools project: https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/src/FSharp.Editing/Common/TypedAstUtils.fs
+//Original code from VisualFSharpPowerTools project: https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/src/FSharp.Editing/Common/TypedAstUtils.fs
```

Comment:
I was getting warnings that the /// on a namespace wasn't valid.  Not sure if I believe it or not.

Reply:
yeah, it's not actually valid :) you gotta do `<namespacedoc>` inside a random element inside the namespace to get that. https://github.com/fsharp/fslang-design/blob/main/tooling/FST-1031-xmldoc-extensions.md#namespacedoc